src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -466,6 +469,9 @@ public static boolean downloadDistribution(PrintStream printStream, String distr
                         OSUtils.getUserAgent(getCurrentBallerinaVersion(), ToolUtil.getCurrentToolsVersion(),
                                 distributionType));
                 conn.setRequestProperty("Accept", "application/json");
+                if (TEST_MODE == true) {
```

Comment:
If we are already passing the `testFlag` , do we really need to keep track of an environment variable as well ?